app/models/solidus_bolt/gateway.rb

```diff
@@ -21,8 +21,28 @@ def authorize(_amount, payment_source, gateway_options)
       ActiveMerchant::Billing::Response.new(false, e, {})
     end
 
-    def capture(_float_amount, _response_code, _gateway_options)
-      raise NotImplementedError, 'capture method has not been implemented in SolidusBolt::Gateway class'
+    def capture(float_amount, response_code, gateway_options)
+      payment = gateway_options[:originator]
+      payment_source = payment.source
+      payment_method = payment.payment_method
+      currency = gateway_options[:currency]
+
+      raise 'Payment has not been authorized yet' unless can_capture?(response_code, payment_method)
```

Comment:
`capture` methods on other extension doesn't make this check here, probably because it's already executed under `can_capture?` condition

Reply:
Removed the check
[21d5259](https://github.com/nebulab/solidus_bolt/commit/21d52591df40a22f4b3b28704bab3c75c9e6d2e1)